Fix minor typo on profile-api.md (#1963)
segmentio/segment-docs: src/personas/profile-api.md

  `https://profiles.segment.com/v1/spaces/<space-id>/collections/users/profiles/<user_identifier>/events`

> - **External IDs**: Query all of a user's external IDs (for example, `anonymous_id`You can query all of a user's external ids (anonymous_id, user_id, ios.):

Reviewer: ```suggestion
- **External IDs**: You can query all of a user's external IDs such as `anonymous_id`, `user_id`.
```
